saleor/app/installation_utils.py

```diff
@@ -60,6 +60,7 @@ def install_app(
             view=extension_data.get("view"),
             type=extension_data.get("type"),
             target=extension_data.get("target"),
+            open_as=extension_data.get("open_as") or AppExtensionOpenAs.POPUP,
```

Comment:
Maybe?
```suggestion
            open_as=extension_data.get("open_as",  AppExtensionOpenAs.POPUP),
```

Reply:
In case when extension_data will have key `open_as` with `None` this will break